Neos.ContentRepository.Core/Classes/Feature/Common/ConstraintChecks.php

```diff
@@ -100,7 +100,7 @@ protected function requireDimensionSpacePointToExist(DimensionSpacePoint $dimens
     protected function requireNodeType(NodeTypeName $nodeTypeName): NodeType
     {
         try {
-            return $this->getNodeTypeManager()->getNodeType($nodeTypeName->value);
+            return $this->getNodeTypeManager()->getNodeType($nodeTypeName);
```

Comment:
the `->value` cast ist just unnecessary as we _do_ accept the dto direct and should use it for type safety